### DISCUSSION TITLE
image won't load/render in github-page

### DISCUSSION THREAD

**Discussion by akhmadgibran** [community/community#133889] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hellow 👋

my problem :
my image .jpg and .png won't load in github-page. but look fine on live preview

i have tried :
1. checking for case sensitive
2. using "./" and "../" 
3. originally the img folder is at root folder, then i tried move it to dist too.

github repo : [link](https://github.com/lieeh/hospitalRework)

github-page deploy : [link](https://lieeh.github.io/hospitalRework/)

**img link code is in index.html file inside docs folder**

this is my 1st time asking in this community, if i have done something wrong, pls tell me. thank you! ✌

screenshot of the live preview :

![screencapture-127-0-0-1-5500-docs-index-html-2024-07-29-13_39_29](https://github.com/user-attachments/assets/8cbee2c0-61d0-4222-85a8-1ab8e30d757d)

**Comment by subhanjandas** [comment, marked as answer]
Hi there! 👋

It looks like you're having issues with your images not loading on your GitHub Pages site even though they appear fine in live preview. 

After going through your given repo, it seems like the images are located in `dist/img` while your `index.html` is in the `docs` folder. 
The paths to the images in your HTML file need to be relative to the docs folder. 

Thanks,

**Reply by akhmadgibran** [reply]
it work! thank you very much!

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @lieeh , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by akhmadgibran** [reply]
oh! im sorry i have made a mistake, i will do better in the future, thanks!!
